library/x509_crt.c

```diff
@@ -58,6 +58,10 @@
 
 #if defined(MBEDTLS_HAVE_TIME)
 #if defined(_WIN32) && !defined(EFIX64) && !defined(EFI32)
+#define WIN32_LEAN_AND_MEAN
+#ifndef _WIN32_WINNT
+#define _WIN32_WINNT 0x0600
```

Comment:
How did you decide on this version?

Reply:
That is the minimum Windows version which provides `inet_pton()` and `InetPton()`
https://learn.microsoft.com/en-us/windows/win32/api/ws2tcpip/nf-ws2tcpip-inet_pton
> The InetPton function is supported on Windows Vista and later.
:rocket: